modules/govuk_jenkins/manifests/jobs/transition_import_hits.pp

```diff
@@ -9,10 +9,25 @@
 #
 class govuk_jenkins::jobs::transition_import_hits(
   $s3_bucket = '',
+  $app_domain = hiera('app_domain'),
```

Comment:
This looks like it's unused.

Reply:
Yeah... you're correct. I put it in so the comment about the `job_url` made more sense and for consistency with the other jobs. Do you think it's better to just not add it?